content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,164 @@
+=======
+Belgium
+=======
+
+.. _belgium/introduction:
+
+Fiscal certification: POS restaurant
+====================================
+
+In Belgium, the owner of a business that offers cooking services (such as a restaurant, a
+food-truck, etc.) and has a turnover (excluding VAT, drinks and take-away food) that exceeds 25 000
+euros a year, is required, by law, to issue their cash register receipts using a
+government-certified **Cash Register System**.
+
+This government-certified system entails the use of a **certified POS system**, along with a device
+called a **Fiscal Data Module** (or **black box**) and a **VAT Signing Card** (delivered after
+registration by the Service Public Federal Finances).
+
+A certified POS system
+----------------------
+
+To ensure full compliance with the law, the Odoo POS system is certified for `Odoo 9 (Odoo Online)
+<https://www.systemedecaisseenregistreuse.be/systemes-certifies>`_, and Odoo 13 to Odoo 15 (**Odoo
+Online** and **odoo.sh**).
+
+A certified POS system must adhere to rigorous government regulations, which means it operates
+differently from a non-certified POS.
+
+- You cannot:
+
+  - Make a refund.
+  - Set up and use the **global discounts** feature (the `pos_discount` module is blacklisted and
+    cannot be activated).
+  - Set up and use the **loyalty programs** feature (the `pos_loyalty` module is blacklisted and
+    cannot be activated).
+  - Reprint receipts (the `pos_reprint` module is blacklisted and cannot be activated).
+  - Modify prices in order lines.
+  - Modify or delete order lines in POS orders.
+  - Sell products without a valid VAT number.
+  - Use a POS that is not connected to an IoT box.
```

Comment:
Actually you can use any pos config that are not connected to an IoT Box. 
But since no black box will be linked to this specific pos, there won't be any signature. 

Another thing to know is that once a pos has been using a black box, it cannot be use anymore without blackbox.

Reply:
what do you mean by "any" pos config? I mean, can you or can you not install these modules? (legally, not in practice, but legally so you don't have to pay a big fine afterward)

Reply:
I should add a warning about that "Another thing to know is that once a pos has been using a black box, it cannot be use anymore without black-box."

Reply:
> what do you mean by "any" pos config? I mean, can you or can you not install these modules? (legally, not in practice, but legally so you don't have to pay a big fine afterward)

Legally you cannot sell anything without blackbox if your restaurant required it, so you shouldn't use any other pos config.
But we can imagine a restaurant and a shop in the same company. You don't want to block the shop with a blackbox, so you can open another pos config without blackbox.

Reply:
Ok, I see, well I could add this as a note for anyone who wants to use another POS